app/upload/views.py

```diff
@@ -51,6 +54,7 @@ def upload_document(service_id):
             ),
             'mlwr_sid': sid,
             'filename': filename,
+            'sending_method': sending_method,
```

Comment:
Do we need to pass this to the frontend URL?

Reply:
no, because the fallback for any sending_method != 'attach' is the standard S3 path / key generation, it's outside of the cycle of needing a sending_method at that point